.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -42,6 +42,24 @@ body:
     validations:
       required: true
 
+  - type: dropdown
+    id: affectVersions
+    attributes:
+      label: Gluten version
+      multiple: true
+      description: Please provide the gluten version in your environment.
+      options:
+        - Gluten-1.0.0
+        - Gluten-1.1.0
+        - Gluten-1.1.1
+        - Gluten-1.2.0
+        - Gluten-1.2.1
+        - Gluten-1.3.0
+        - Gluten-1.4.0
+        - UnReleased
```

Comment:
Maybe, better to reverse the order to show latest active version at the top, and remove the 3rd release digit since minor releases have small differences.

```
- main branch
- Gluten-1.4
- Gluten-1.3
- Gluten-1.2
- Gluten-1.1
- Gluten-1.0
```